CHANGELOG.md

```diff
@@ -1,5 +1,152 @@
 # Release Notes
 
+## Version 1.31.0 - 2023-07-17
+
+Features:
+* Add back VMware driver support [#16796](https://github.com/kubernetes/minikube/pull/16796)
+* `docker-env` supports the containerd runtime (experimental) [#15452](https://github.com/kubernetes/minikube/pull/15452) [#16761](https://github.com/kubernetes/minikube/pull/16761)
+* Automatically renew expired kubeadm certs [#16249](https://github.com/kubernetes/minikube/pull/16249)
+* New addon inspektor-gadget [#15869](https://github.com/kubernetes/minikube/pull/15869)
+
+Major Improvements:
+* VM drivers: Fix all images getting removed on stop/start (40% start speedup) [#16655](https://github.com/kubernetes/minikube/pull/16655)
+* Addon registry: Add support for all architectures [#16577](https://github.com/kubernetes/minikube/pull/16577)
+* QEMU: Fix failing to interact with cluster after upgrading QEMU version [#16853](https://github.com/kubernetes/minikube/pull/16853)
+* macOS/QEMU: Auto unblock bootpd from firewall if blocking socket_vmnet network [#16714](https://github.com/kubernetes/minikube/pull/16714) [#16789](https://github.com/kubernetes/minikube/pull/16789)
+* `minikube cp` supports providing directory as a target [#15519](https://github.com/kubernetes/minikube/pull/15519)
+
+Minor Improvements:
+* Always use cni unless running with dockershim [#14780](https://github.com/kubernetes/minikube/pull/14780)
+* none driver: Check for CNI plugins before starting cluster [#16419](https://github.com/kubernetes/minikube/pull/16419)
+* QEMU: Add ability to create extra disks [#15887](https://github.com/kubernetes/minikube/pull/15887)
+* --kubernetes-version: Assume latest patch version if not specified [#16569](https://github.com/kubernetes/minikube/pull/16569)
+* audit: Set default max file size [#16543](https://github.com/kubernetes/minikube/pull/16543)
+* service: Fail if no pods available [#15079](https://github.com/kubernetes/minikube/pull/15079)
+* Kicbase: Image building uses docker buildx [#16252](https://github.com/kubernetes/minikube/pull/16252)
```

Comment:
nit: users dont know whats kicbase, maybe replace with
docker/podman drivers: use buildx for `image build` command